dev-libs/Ice/Ice-3.5.1.ebuild

```diff
@@ -161,6 +173,22 @@ src_compile() {
 		emake -C cpp/doc || die "building docs failed"
 	fi
 
+	if use php; then
+		local slot
+		for slot in $(php_get_slots); do
```

Comment:
Are you sure these are the right parentheses? This is not make :stuck_out_tongue_winking_eye: